tests/release/release.go

```diff
@@ -27,6 +28,12 @@ const (
 
 	avgPipelineCompletionTime = 10 * time.Minute
 	defaultInterval           = 100 * time.Millisecond
+
+	Failure1ApplicationName       = "test"
```

Comment:
All the names could be reused just by deleting and creating the namespaces at the beginning and at the end of each tests. Also, maybe this could be moved to a different file even.

Reply:
Agreed, and this is similar to Kasem's comment. I think the approach should be one in that we aren't using `FailureX` to differentiate